deploy/kube-config/kafka/heapster-service.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: v1
```

Comment:
Yes, the `yaml` file supports  delimiter `---` to define a block.
So we can put all configs in a single file, separate with `---`.Like:

```
---
apiVersion: v1
kind: ReplicationController
metadata:
  name: kafka
 ....

---
apiVersion: v1
kind: Service
metadata:
  labels:

```